recipes/spix/all/patches/0001-use-conan-libs-0.5.patch

```diff
@@ -0,0 +1,32 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,7 +5,6 @@ option(SPIX_BUILD_EXAMPLES "Build Spix examples." ON)
+ option(SPIX_BUILD_TESTS "Build Spix unit tests." OFF)
+ set(SPIX_QT_MAJOR "6" CACHE STRING "Major Qt version to build Spix against")
+ 
+-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/cmake/modules")
+ set(CMAKE_CXX_STANDARD 17)
+ 
+ # Hide symbols unless explicitly flagged with SPIX_EXPORT
+diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
+index 723de5e..f234bec 100644
+--- a/lib/CMakeLists.txt
++++ b/lib/CMakeLists.txt
+@@ -8,7 +8,7 @@ include(CMakePackageConfigHelpers)
+ # Dependencies
+ #
+ find_package(Threads REQUIRED)
+-find_package(AnyRPC REQUIRED)
++find_package(anyrpc REQUIRED)
```

Comment:
This can be set from the recipe on CMakeDeps with https://blog.conan.io/2022/12/20/New-conan-release-1-55.html